.github/scripts/prepare-image-tags.js

```diff
@@ -0,0 +1,23 @@
+module.exports = () => {
+    const { CURRENT_NODE, LATEST_NODE, RELEASE_TAG, IMAGE_NAME, FRAMEWORK_VERSION } = process.env
+    const tags = [];
+
+    if (CURRENT_NODE === LATEST_NODE) {
+        tags.push(`${IMAGE_NAME}:${RELEASE_TAG}`);
+    }
+
+    if (RELEASE_TAG === "latest") {
+        // latest version
+        const latestTagName = FRAMEWORK_VERSION ? `${IMAGE_NAME}:${CURRENT_NODE}-${FRAMEWORK_VERSION}` : `${IMAGE_NAME}:${CURRENT_NODE}`
```

Comment:
Would look better as:

```js
const latestTagName = FRAMEWORK_VERSION
    ? `${IMAGE_NAME}:${CURRENT_NODE}-${FRAMEWORK_VERSION}`
    : `${IMAGE_NAME}:${CURRENT_NODE}`
```

In a single line, it breaks weirdly in GitHub review (too many colons 😄 )
<img width="921" alt="image" src="https://user-images.githubusercontent.com/23726914/117265874-fa043e00-ae54-11eb-932d-c9a45816e522.png">